_styles/-theme.scss

```diff
@@ -3,8 +3,8 @@
 
 // colors
 [data-dark="false"] {
```

Comment:
Here's another case where it wasn't safe to to `theirs` on all `_styles`. Probably want to keep your "Greene green" here, and not the template default light blue.